lib/mechanize/test_case/infinite_refresh_servlet.rb

```diff
@@ -4,7 +4,6 @@ def do_GET(req, res)
     res['Content-Type'] = req.query['ct'] || "text/html"
     res.status = req.query['code'] ? req.query['code'].to_i : '302'
     number = req.query['q'] ? req.query['q'].to_i : 0
-    res['Refresh'] = "0;url=http://#{address}/infinite_refresh?q=#{number + 1}\r\n";
+    res['Refresh'] = "0;url=http://#{address}/infinite_refresh?q=#{number + 1}";
```

Comment:
is this (and the above) what you’re considering the internals? Seems like part of the spec, no?

Reply:
No. The first commit is what I was considering the internals. The second commit, which removes the CR/LF was to fix additional failures I encountered when I went to put this PR together off master. I'm not sure what changed as I didn't need to do this last time. It sounded to me like those characters are no valid in HTTP headers though.